modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -158,31 +101,27 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: kubevirt-vm-latency-checkup-config
-  namespace: <target_namespace> <1>
 data:
-  spec.image: registry.redhat.io/container-native-virtualization/vm-network-latency-checkup:v4.12.0
   spec.timeout: 5m
   spec.serviceAccountName: vm-latency-checkup-sa
-  spec.param.network_attachment_definition_namespace: <target_namespace> <2>
-  spec.param.network_attachment_definition_name: "bridge-network" <3>
-  spec.param.max_desired_latency_milliseconds: "10" <4>
-  spec.param.sample_duration_seconds: "5" <5>
-  spec.param.source_node: "worker1" <6>
-  spec.param.target_node: "worker2" <7>
-----
-<1> The namespace where the checkup is to be executed. This must be an existing namespace where the `NetworkAttachmentDefinition` object resides.
-<2> The namespace where the `NetworkAttachmentDefinition` object resides.
-<3> The name of the `NetworkAttachmentDefinition` object.
-<4> Optional: The maximum desired latency, in milliseconds, between the virtual machines. If the measured latency exceeds this value, the checkup fails.
-<5> Optional: The duration of the latency check, in seconds.
-<6> Optional: When specified, latency is measured from this node to the target node. If the source node is specified, the `spec.param.target_node` field cannot be empty.
-<7> Optional: When specified, latency is measured from the source node to this node.
+  spec.param.network_attachment_definition_namespace: "default"
```

Comment:
The NetworkAttachmentDefinition is expected to be at the same namespace as the checkup.
```suggestion
  spec.param.network_attachment_definition_namespace: <target_namespace>
```